src/com/projectkorra/projectkorra/command/ToggleCommand.java

```diff
@@ -66,7 +68,7 @@ public void execute(final CommandSender sender, final List<String> args) {
 				bPlayer.toggleBending();
 			}
 		} else if (args.size() == 1) {
-			if (args.size() == 1 && args.get(0).equalsIgnoreCase("all") && this.hasPermission(sender, "all")) { // bending toggle all.
+			if (args.get(0).equalsIgnoreCase("all") && this.hasPermission(sender, "all")) { // bending toggle all.
```

Comment:
`args.get(0)` is used 13 times in this if/else, maybe make a local variable for it with a nice name?

Also maybe consider one for `args.get(0).split("Passives")[0]` because that's quite a long statement that you do four times and the purpose isn't obvious.